src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/AuthDetails.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services;
+
+import uk.gov.hmcts.reform.idam.client.models.UserDetails;
+
+public class AuthDetails {
```

Comment:
how about we move AuthDetails to the idam client?

Reply:
It seems that this mostly relevant only to call the Ccd Api.
It would seems sensible to move it when its used like this somewhere else ?

Reply:
every project will need this information

Reply:
Ok Ive composed this with UserAuthDetails and AuthDetails needed for our ccd call. 
We can move this in a different PR to another repository 😄